docs/source/host-collect-analyze/dns.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Host DNS
+description: Collect DNS queries on the host machine.
+---
+
+The `dns` host collector can be used to help diagnose DNS resolution problems on the host machine. During execution, the collector performs various DNS `A` record queries to troubleshoot DNS resolution. It does the following:
+
+- Reads the contents of `/etc/resolv.conf`
+- Performs DNS lookups for specified hostnames
+- Outputs query results including IP addresses and the result of the DNS lookup
+
+## Parameters
+
+In addition to the [shared collector properties](https://troubleshoot.sh/docs/collect/collectors/#shared-properties), the `dns` host collector accepts the following parameters:
+
+##### `hostnames` (Optional)
+
+A list of hostnames to query. These can include both resolvable domains and non-resolvable domains to test various scenarios (e.g., wildcard DNS). Defaults to `["*"]`.
+
+## Example Collector Definition
+
+```yaml
+apiVersion: troubleshoot.sh/v1beta2
+kind: SupportBundle
+metadata:
+  name: sample
+spec:
+  hostCollectors:
+    - dns:
+        hostnames:
```

Comment:
let's add a collectorName here too, so we have the option to have an explanatory analyzer example too